go-controller/pkg/libovsdbops/router.go

```diff
@@ -1140,7 +1140,8 @@ func DeleteNATsWithPredicateOps(nbClient libovsdbclient.Client, ops []libovsdb.O
 				router.Nat = extractUUIDsFromModels(&deleted)
 				natUUIDs.Insert(router.Nat...)
 			},
-			BulkOp: true,
+			BulkOp:      true,
+			ErrNotFound: false,
```

Comment:
this is implicit as false is the default value for a boolean. But you are adding it here to make it explicit and consistent with the others, right?

Reply:
to be explicit yeah